src/views/v2/month/mobile-events/mobile-day/day-marker.php

```diff
@@ -11,8 +11,21 @@
  *
  * @version 4.9.4
  *
+ * @var string $day_date Date for this marker, following `Y-m-d` format.
+ *
  */
+
+if ( ! isset( $day_date ) ) {
+	return;
+}
+
+$day_date_datetime = strtotime( $day_date );
```

Comment:
This conversion, from a `Y-m-d` to the timestamp, will be dependant on the timezone of the server.
I feel here we might want it to be dependant on the **site** timezone.
I suggest we use `Dates::build_date_object( $day_date )->getTimestamp()` and `use Tribe__Dates_Utils as Dates` at the top of the file.

Reply:
;)